### ISSUE TITLE
create should allow to inherit options of the parent

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#1490]
would be nice if `create` allowed to inherit from its superdataset
- `.gitattributes` either it would be a "template" (as discussed in #1462) or smth else
- 'annex wanted' settings

**Comment by mih** [comment]
Yes. If you can dump the list of config variables relevant for this change in here, I should be able to get this done.
